@xen-orchestra/lite/src/components/TabConsole.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import { withState } from 'reaclette'
+
+import Button from './Button'
+
+import Console, { IConsole } from './Console'
+
+interface ParentState {}
+
+interface State {
+  console: React.RefObject<IConsole>
+}
+
+interface Props {
+  vmId: string
+}
+
+interface ParentEffects {}
+
+interface Effects {}
+
+interface Computed {}
+
+const TabConsole = withState<State, Props, Effects, Computed, ParentState, ParentEffects>(
+  {
+    initialState: () => ({
+      console: React.createRef(),
+    }),
+  },
+  ({ state, vmId }) => (
+    <div style={{ height: '100vh' }}>
+      {state.console.current !== null &&
+        <Button label='CTRL+ALT+DEL' onClick={state.console.current._effects.sendCtrlAltDel} />
```

Comment:
@julien-f What do you think of this? Would there be an easier way for a parent to call its Reaclette children's effects?

Reply:
You could pass an function to the children to send the effect to the parent.

Otherwise no, I have no better ideas.

Reply:
Ok, so let's do it like this @MathieuRA: pretty much like you were sending the RFB to the parent in your previous implementation, but only sending the effect instead, so that the parent doesn't have to know what a RFB is and how it works.